README.md

```diff
@@ -563,7 +563,13 @@ class Post < ActiveRecord::Base
 end
 ```
 
-This allows you to store each model's versions in a separate table, which is useful if you have a lot of versions being created.
+### Advantages
+
+1. For models which have a lot of versions, storing each model's versions in a
+   separate table can improve the performance of certain database queries.
+1. Store different version [metadata](#storing-metadata) for different models.
```

Comment:
Heh, that's a markdown thing. :D  You don't have to increment your numbers, md will do it for you.  Using all ones makes long lists easy to reorder.